script.js

```diff
@@ -8,30 +8,69 @@ class AngajatIT {
         this.departament= departament;
     }
     afiseazaVarsta(){
-        console.log('Neimplementat');
+        const anulNasterii = +this.CNP.slice(1,3);
+        const dataCurenta = new Date();
+        const currentYear = dataCurenta.getFullYear();
+        let currentAge = String(currentYear - anulNasterii).slice(-2);
+
+        const currentDay = dataCurenta.getDate();
+        const dayFromCNP = Number(this.CNP.slice(5,7));
+        const currentMonth = String(dataCurenta.getMonth() + 1).padStart(2, '0');
+        const monthFromCNP = this.CNP.slice(3, 5);
+
+        if(monthFromCNP >  currentMonth){
+            currentAge--;
+        }else if(monthFromCNP < currentMonth){
+            currentAge;
+        }else if(monthFromCNP === currentMonth){
+            if(currentDay > dayFromCNP){
+                currentAge;
+            }else if(currentDay < dayFromCNP){
+                currentAge--;
+            }
+        }
+     
     }
     afiseazaAnulAngajarii(){
         console.log('Neimplementat');
+        const currentYear = (new Date().getFullYear());
+        const yearEmployee = this.vechime;
+        const employeeSeniority = currentYear - yearEmployee;
```

Comment:
puteai sa pui direct aici currentYear- this.vechime

Reply:
am modificat